app/packages/state/src/hooks/useSchemaSettings.ts

```diff
@@ -35,14 +37,16 @@ const skipFiled = (path: string, ftype: string) => {
     path.endsWith(".bounding_box")
   );
 };
-const disabledField = (path: string, ftype: string, subfield: string) => {
+const disabledField = (path: string, ftype: string) => {
   return (
     path === "tags" ||
     path === "filepath" ||
     (ftype === OBJECT_ID_FIELD && path === "id") ||
     path.startsWith("metadata") ||
     path.endsWith("frames.frame_number") ||
-    ftype === FRAME_NUMBER_FIELD
+    ftype === FRAME_NUMBER_FIELD ||
+    path === "group" ||
+    path === "frames.group"
```

Comment:
I don't think this actually covers the general case case of group datasets. I suppose dynamic groups (as views) complicates things more. Ideally there is a way to inspect a required group field in the schema cc @brimoor

Reply:
Got it fixed thanks to you! it is looking at the dataset.group_field now instead